dat.go

```diff
@@ -75,6 +75,7 @@ var (
 	globalincref bool
 	seq          int
 	maxtab       uint // size of a tab, in units of the '0' character
+	texpand      bool // defines whether to expand tab to spaces
```

Comment:
Nit: can you call this tabexpand? It will make easier for (at least me) to remember what it's for later.

Reply:
That makes sense. Just updated my PR with this change.